search/interfaces/graphql/dto/dto.go

```diff
@@ -7,6 +7,7 @@ type CommerceSearchRequest struct {
 	SortBy          string
 	SortDirection   string
 	KeyValueFilters []CommerceSearchKeyValueFilter
+	Query           *string
```

Comment:
Just out of curiosity, any specific reason why a pointer is used here? To enable nil setting?

Reply:
Good catch. I think it was done because of that, but it's not necessary because the search does not expect a pointer anyway.

Reply:
I think it was because the graphql lib suggested this - if graphql model mapping works without pointer then lets change this.

Reply:
Ah okay! I checked if it is working without the pointer and seems like it does :)